RecoBTag/TensorFlow/src/sorting_modules.cc

```diff
@@ -1,5 +1,5 @@
 
```

Comment:
make sure that changes here are matching the changes made in #23335 so that we don't end up with multiple sorting_modules files